src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public enum Color {
+     White,
+     Black,
+     Red,
+     Yellow,
+     Orange,
+     Green,
+     Blue,
+     Purple,
+     Pink,
+     Brown,
+     Grey,
```

Comment:
**Don’t forget how to name constants according to style guide (the same with naming of enum values)**
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-forget-how-to-name-constants-according-to-style-guide-the-same-with-naming-of-enum-values)